### PR TITLE
Flags lists the JUJU_DEV_FEATURE_FLAGS (if any) that the environment uses

### PR DESCRIPTION
**Pull request by marcoceppi**
simple plugin

```
$ export JUJU_DEV_FEATURE_FLAGS="storage jes"
$ juju bootstrap
$ unset JUJU_DEV_FEATURE_FLAGS
$ juju flags
storage jes
```

### CODE REVIEW

File: juju-flags [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+if [ "$1" == '--help' ]; then
+    echo "juju flags"
+    echo ""
+    echo "List flags used in an environment"
+    exit 0
+fi
+
+# Check for upstart, otherwise check process
+juju ssh 0 "if [ -f /etc/init/jujud-machine-0.conf ]; then grep JUJU_DEV_FEATURE_FLAGS /etc/init/jujud-machine-0.conf | awk -F'\"' '{ print \$2 }'; else ps -ae -o command= | grep [j]ujud | grep JUJU_DEV_FEATURE_FLAGS | awk -f'\"' '{ print \$2 }'; fi" 2>/dev/null
```
Comment (chuckbutler) [inline juju-flags]: Seems a bit hairy of a one liner, but it works. +1

Comment (marcoceppi) [inline juju-flags]: Giant if statements piped over SSH, what's the worst that could happen ;)

Comment (chuckbutler) [inline juju-flags]: yeah but its responding with data, not altering stuff... i'll allow it :8ball:

**Comment by chuckbutler** [comment]
+1 LGTM - works as advertised.

**Pull request merged**